omnibus/config/templates/updater/datadog-agent-exp.service.erb

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=Datadog Agent Experiment
+After=network.target
+OnFailure=datadog-agent.service 
+Conflicts=datadog-agent.service 
+Before=datadog-agent.service 
+JobTimeoutSec=3000
+Wants=datadog-agent-trace-exp.service datadog-agent-process-exp.service datadog-agent-sysprobe-exp.service datadog-agent-security-exp.service
```

Comment:
Could we use `Requires` here to avoid having to stop all dependent services manually when stopping the experiment?

Reply:
Good point, I'll test that change after when I add end to end tests on systemd